src/redux/store.ts

```diff
@@ -8,3 +9,7 @@ export const store = configureStore({
   reducer: rootReducer,
   devTools: process.env.NODE_ENV === "development",
 })
+
+export type RootState = ReturnType<typeof store.getState>
+
+export type AppDispatch = typeof store.dispatch
```

Comment:
create types file inside redux folder and move this type here

Reply:
`types.ts` file has been created in `redux` folder and `AppDispatch` type moved into it.